dist/stylekit.js

```diff
@@ -660,6 +660,7 @@ const removeToast = action(toastStore, 'removeToast', (store, toastId) => {
   const existingToasts = store.get();
   store.set(existingToasts.filter(toast => toast.id !== toastId));
 });
+const DELAY_BEFORE_REMOVING_TOAST = 175;
```

Comment:
Hmm it's uppercase in dist but pascal case in src. Maybe we should add a pre-commit `yarn build` step?

Reply:
Pre-push might be better I think.

Reply:
Let's go with just removing dist from git actually, I think the CI handles this.